src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -192,9 +192,40 @@ static boolean hasOcr(CaseDetails theCase) {
             .map(validation -> validation
                 .mapError(throwable -> "Invalid journeyClassification. Error: " + throwable.getMessage())
             )
+            .map(validation -> validateClassification(validation, theCase))
             .orElse(invalid("Missing journeyClassification"));
     }
 
+    private static Validation<String, Classification> validateClassification(
+        Validation<String, Classification> validation,
+        CaseDetails theCase
+    ) {
+        if (validation.isInvalid()) {
+            return validation;
+        }
+        Classification classification = validation.get();
+        if (SUPPLEMENTARY_EVIDENCE.equals(classification)) {
+            return invalid(format(
+                "Event %s not allowed for the current journey classification %s",
+                "createCase",
+                classification
+            ));
+        }
+
+        if (EXCEPTION.equals(classification)) {
+            Optional<List<Map<String, Object>>> ocrDataOpt = getOcrData(theCase);
+            if (!ocrDataOpt.isPresent() || ocrDataOpt.get().isEmpty()) {
```

Comment:
Looks similar as `hasOcrData`. Can we reuse it? after that it's nicer to join both ifs into 1